src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -1447,50 +1449,68 @@ def update_site_appsettings(stack, site_config):
     def _load_stacks(self):
         if self._stacks:
             return
-        raw_list = self._client.provider.get_available_stacks()
-        stacks = raw_list['value']
-        config_mappings = {
-            'node': 'WEBSITE_NODE_DEFAULT_VERSION',
-            'python': 'python_version',
-            'php': 'php_version',
-            'aspnet': 'net_framework_version'
-        }
-
+        os_type = ('Linux' if self._linux else 'Windows')
+        raw_stacks = self._client.provider.get_available_stacks(os_type_selected=os_type, raw=True)
+        bytes_value = raw_stacks._get_next().content
+        json_value = bytes_value.decode('utf8')
+
+        import json
```

Comment:
Per the previous comment, this should be placed at the beginning of the file. The only reason for an import statement to be placed in the middle of a function is for the performance reason (to delay module load). The `json` module doesn't stands out and it is already loaded elsewhere. Please move it to the head of the file.

Reference:
- https://www.python.org/dev/peps/pep-0008/#imports
- https://google.github.io/styleguide/pyguide.html?showone=Imports_formatting#Imports_formatting